xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/job/question/EntitySelectionTest.java

```diff
@@ -75,4 +75,16 @@ public void compareTo()
         EntitySelection bEntitySelection = new EntitySelection(bEntityReference);
         assertEquals(-1, aEntitySelection.compareTo(bEntitySelection));
     }
+
+    @Test
+    void compareToEntityReferenceNull() {
```

Comment:
code style, should be on next line.

Reply:
Fixed https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-5b98cea563c1080c7d45682b954847edR80